gateway-js/src/loadSupergraphSdlFromStorage.ts

```diff
@@ -89,6 +200,18 @@ export async function loadSupergraphSdlFromStorage({
       );
     }
   } else {
+    try {
+      await submitOutOfBandReport({
+        error: fetchErrorMsg + result.status + ' ' + result.statusText,
+        request,
+        response: result,
+        startedAt: startTime.toISOString(),
+        endedAt: endTime.toISOString(),
+        fetcher
+      });
+    } catch(e) {
+      throw new Error(e);
+    }
```

Comment:
This `try` / `catch` doesn't seem necessary as opposed to just calling it, right? (It's just re-throwing the trapped error.)

```suggestion
    await submitOutOfBandReport({
      error: fetchErrorMsg + result.status + ' ' + result.statusText,
      request,
      response: result,
      startedAt: startTime.toISOString(),
      endedAt: endTime.toISOString(),
      fetcher
    });
```